test/src/main/java/org/apache/accumulo/test/compaction/ExternalCompactionMetricsIT.java

```diff
@@ -127,7 +127,7 @@ public void testMetrics() throws Exception {
             if (shutdownTailer.get()) {
               break;
             }
-            if (s.startsWith(MetricsProducer.METRICS_MAJC_QUEUED)) {
+            if (s.startsWith(TSERVER_MAJC_QUEUED.getName())) {
```

Comment:
Is this equivalent to what it was previously?

Reply:
Good catch. it is very confusing having two metrics with such similar names.